### DISCUSSION TITLE
GO:0120326 dbxref

### DISCUSSION THREAD

**Discussion by stephanielipow** [geneontology/helpdesk#407] [Q&A] [answered]
Not sure if this is where to report it, but I think this should be hasDbXref instead of hasExactSynonym:

   
...
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/GO_0120326"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasExactSynonym"/>
        <owl:annotatedTarget>appressorium-mediated penetration into host</owl:annotatedTarget>
        **<oboInOwl:hasExactSynonym>GOC:ach</oboInOwl:hasExactSynonym>**
    </owl:Axiom>

obo:
[Term]
id: GO:0120326
name: appressorium-mediated entry into host
namespace: biological_process
def: "Penetration by a symbiont into a host organism via an appressorium. The host is defined as the larger of the organisms involved in a symbiotic interaction." [GOC:ach, GOC:krc, PMID:14731267, PMID:22589729]
synonym: "appressorium-mediated host invasion" EXACT [GOC:ach]
synonym: "appressorium-mediated host penetration" EXACT [GOC:ach]
synonym: "appressorium-mediated invasion into host" EXACT [GOC:ach]
**synonym: "appressorium-mediated penetration into host" EXACT [] {EXACT="GOC:ach"}**
is_a: GO:0044409 ! entry into host
property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/21505 xsd:anyURI
created_by: krc
creation_date: 2021-08-04T00:48:35Z

**Comment by suzialeksander** [comment]
@kltm this should be bumped to a specific repo but I'm not sure which one

**Comment by kltm** [comment, marked as answer]
@suzialeksander This looks like an ontology change to me--moving it over there.

**Reply by kltm** [reply]
Now https://github.com/geneontology/go-ontology/issues/23868
